### PR TITLE
Fix constraint type issue

### PR DESCRIPTION
**Pull request by ArcturusZhang**
Fixes https://github.com/Azure/azure-sdk-for-go/issues/13604

### CODE REVIEW

**Comment by ArcturusZhang** [comment]
Hi @jhendrixMSFT could you please take a look at this PR?

File: src/Extensions.cs [hunk 1]
```diff
@@ -717,13 +717,17 @@ public static string GetConstraint(string name, string constraintName, string co
...
                 {
-                    value = $"float64({constraintValue})";
-                }
-                else
-                {
-                    value = $"int64({constraintValue})";
+                    case "float64":
+                        value = $"float64({constraintValue})";
+                        break;
+                    case "int32":
```
Comment (jhendrixMSFT) [inline src/Extensions.cs:725]: If you're adding `int32` you might as well add `float32` above.

Comment (ArcturusZhang) [reply src/Extensions.cs:725]: Well, in [this file](https://github.com/Azure/autorest.go/blob/cd02b097dc9a7ba3d77cdbc466882c7ceb2bc91c/src/Model/PrimaryTypeGo.cs#L60) we do not support `float32` in the SDK, therefore I suppose maybe we could also omit this one?

Comment (jhendrixMSFT) [reply src/Extensions.cs:725]: Good point, we can omit it here.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**